README.md

```diff
@@ -72,24 +91,27 @@ Kubernetes scripts depend on Kustomize, so make sure that you have a newer versi
 that supports it (at least 1.14 or above). 
 
 If you do, you can simply run the following command from the `sockshop` directory
-and replace `BACKEND` with one of the following indicating the type of back-end you want to deploy.
+and replace the `SOCKSHOP_BACKEND` variable with one of the following indicating the type of back-end you want to deploy.
```

Comment:
"and set the `SOCKSHOP_BACKEND` variable to one of the following values, indicating the type of back-end you want to deploy."